power-the-polls-form/src/components/state-info/state-info.tsx

```diff
@@ -26,19 +26,20 @@ export class StateInfoComponent {
    @State() public jurisdictions: Array<Jurisdiction> = [];
    @State() public info?: StateInfo;
 
-   public async componentWillLoad() {
+   public componentWillLoad() {
       if( this.stateId ) {
-         this.jurisdictions = await fetchFromWE( `/jurisdictions/?summary=true&state_id=${this.stateId}` );
-         this.info = await fetchFromWE( `/states/${this.stateId}/` );
+         fetchFromWE( `/jurisdictions/?summary=true&state_id=${this.stateId}` )
+            .then( x => this.jurisdictions = x );
+         fetchFromWE( `/states/${this.stateId}/` )
+            .then( x => this.info = x );
       }
```

Comment:
@mojowen So apparently if componentWillLoad returns a promise/async it will wait for it to complete before the first render. Upon consideration I actually like that, but it is not at all documented well enough and ends up as a gotcha.